board/main_comms.h

```diff
@@ -477,6 +477,17 @@ int comms_control_handler(ControlPacket_t *req, uint8_t *resp) {
         bool ret = can_init(CAN_NUM_FROM_BUS_NUM(req->param1));
         UNUSED(ret);
       }
+      break;
+    // *** 0xfd: read logs
+    case 0xfd:
+      if (req->param1 == 1U) {
+        logging_find_read_index();
+      }
+
+      if (logging_read(resp)) {
+        resp_len = sizeof(log_t);
```

Comment:
seems nicer to either pass in resp_len and write it in logging_read or return it from logging_read